bin/main.c

```diff
@@ -160,6 +203,13 @@ on_conn(options_t *opts, int con, int in, int out, const struct addrinfo *ai)
 
       if (opts->psku)
         srv.psk = srv_psk_cb;
+      else if (opts->crtf)
+      {
```

Comment:
Don't wrap the braces here. Also, we don't do unbalanced braces. If the `else if` requires braces, the `if` gets them too.